src/textures/mlx_xpm42.c

```diff
@@ -37,6 +37,17 @@
  * straight forward to this format however.
  */
 
+/**
+ * Parses HEX color channel e.g: "0F"
+ * 
+ * @param channel The 2 character string to parse.
+ * @return Int value of the channel.
+ */
+static uint8_t parse_hex_channel(char *channel) {
+	char temp_chan[] = {channel[0], channel[1], '\0'};
+	return (uint8_t)strtol(temp_chan, NULL, 16);
```

Comment:
I know its technically not necessary but in the Wiki I describe the repo code style, I prefer to keep the return values in brackets: `return (...);`

Reply:
Ah! totally forgot to read the wiki, maybe you should consider mentioning the wiki code style in [CONTRIBUTING.md](https://github.com/W2Codam/MLX42/blob/master/CONTRIBUTING.md)?

Reply:
Good point! Will do.

Reply:
Fixed in: f5f359e